doc/api/child_process.md

```diff
@@ -281,9 +284,11 @@ changes:
 The `child_process.execFile()` function is similar to [`child_process.exec()`][]
 except that it does not spawn a shell by default. Rather, the specified
 executable `file` is spawned directly as a new process making it slightly more
-efficient than [`child_process.exec()`][].
+efficient than [`child_process.exec()`][]. Also `child_process.execFile()` takes
```

Comment:
I think this sentence should be deleted, but if we keep it, let's add in the missing comma:

```suggestion
efficient than [`child_process.exec()`][]. Also, `child_process.execFile()` takes
```